src/main/java/appeng/client/render/items/ItemEncodedPatternRenderer.java

```diff
@@ -34,8 +34,8 @@
 public class ItemEncodedPatternRenderer implements IItemRenderer
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a constructor to the class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1258)